DOCUMENTATION.md

```diff
@@ -0,0 +1,47 @@
+StyleCop.Analyzers provides warnings that indicate style and consistency rule violations in C# code. The warnings are organized into rule areas such as documentation, layout, naming, ordering, readability, spacing, and so forth. Each warning signifies a violation of a style or consistency rule. This section provides an explanation of each of the default StyleCop.Analyzers rules. 
+
+### Rule areas
+**[Special Rules (SA0000-)](documentation/SpecialRules.md)**
+
+Rules which provide special functionality like workarounds, configuration errors, etc.
+
+**[Spacing Rules (SA1000-)](documentation/SpacingRules.md)**
+
+Rules which enforce spacing requirements around keywords and symbols in the code.
+
+**[Readability Rules (SA1100-)](documentation/ReadabilityRules.md)**
+
+Rules which ensure that the code is well-formatted and readable.
+
+**[Ordering Rules (SA1200-)](documentation/OrderingRules.md)**
+
+Rules which enforce a standard ordering scheme for code contents.
+
+**[Naming Rules (SA1300-)](documentation/NamingRules.md)**
+
+Rules which enforce naming requirements for members, types, and variables.
+
+**[Maintainability Rules (SA1400-)](documentation/MaintainabilityRules.md)**
+
+Rules which improve code maintainability.
+
+**[Layout Rules (SA1500-)](documentation/LayoutRules.md)**
+
+Rules which enforce code layout and line spacing.
+
+**[Documentation Rules (SA1600-)](documentation/DocumentationRules.md)**
+
+Rules which verify the content and formatting of code documentation.
+
+### Additional documentation
+**[Configuration](documentation/Configuration.md)**
+
+Describes the configuration options for StyleCop.Analyzers
+
+**[How to enable the configuration](documentation/EnableConfiguration.md)**
+
+Describes how to enable the `stylecop.json` file for usage.
+
+**[Known changes](documentation/KnownChanges.md)**
+
+Describes the known differences between StyleCop.Analyzers and StyleCop.Classic
```

Comment:
:bulb: I've been referring to them as "StyleCop Analyzers" and "StyleCop Classic" (no `.`)